src/js/linter.js

```diff
@@ -156,4 +156,16 @@ class Linter extends EventEmitter {
     }
 }
 
+parser.TreeNodeList.prototype.forEach = function (callback) {
```

Comment:
I put this here because I wasn't sure what the node/common-js way of including this inside of a utility file.